### PR TITLE
Heated cylinders test case

### PR DESCRIPTION
**Pull request by oleburghardt**
This PR adds a heated cylinder case to the tutorials page to showcase conjugate heat transfer, solving heat equations in solids and performing coupled discrete adjoints for multizone problems.

### CODE REVIEW

File: Inc_Heated_Cylinders/flow_cylinder.cfg [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+%
+% Restart flow input file
+SOLUTION_FILENAME= solution_flow.dat
+%
+% Restart adjoint input file
+SOLUTION_ADJ_FILENAME= solution_adj.dat
+%
+% Output file format (TECPLOT, TECPLOT_BINARY, PARAVIEW,
+%                     FIELDVIEW, FIELDVIEW_BINARY)
+OUTPUT_FORMAT= TECPLOT
```
Comment (TobiKattmann) [inline Inc_Heated_Cylinders/flow_cylinder.cfg]: Please change to `OUTPUT_FILES= ( RESTART_ASCII, PARAVIEW, SURFACE_PARAVIEW)` or to the output you prefer... to make it easier you might as well erase it in all the zone configs and just put one in the master

File: Inc_Heated_Cylinders/flow_cylinder.cfg [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+OBJECTIVE_FUNCTION= TOTAL_HEATFLUX
+%
+% List of weighting values when using more than one OBJECTIVE_FUNCTION. Separate by commas and match with MARKER_MONITORING.
+OBJECTIVE_WEIGHT = 1.0
+%
+% Read binary restart files (YES, NO)
+READ_BINARY_RESTART = NO
+%
+% Write binary restart files (YES, NO)
+WRT_BINARY_RESTART = NO
```
Comment (TobiKattmann) [inline Inc_Heated_Cylinders/flow_cylinder.cfg]: This option is deprecated, right? And handled via OUTPUT_FILES

File: Inc_Heated_Cylinders/flow_cylinder.cfg [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+%                                             INVERSE_DESIGN_HEATFLUX, SURFACE_TOTAL_PRESSURE,
+%                                             SURFACE_MASSFLOW, SURFACE_STATIC_PRESSURE, SURFACE_MACH)
+% For a weighted sum of objectives: separate by commas, add OBJECTIVE_WEIGHT and MARKER_MONITORING in matching order.
+OBJECTIVE_FUNCTION= TOTAL_HEATFLUX
+%
+% List of weighting values when using more than one OBJECTIVE_FUNCTION. Separate by commas and match with MARKER_MONITORING.
+OBJECTIVE_WEIGHT = 1.0
+%
+% Read binary restart files (YES, NO)
+READ_BINARY_RESTART = NO
```
Comment (TobiKattmann) [inline Inc_Heated_Cylinders/flow_cylinder.cfg]: oh right... and you probably have to update the `solution_flow_*.dat` files as well as they are currently ASCII format and with the switch to .csv files for ascii that doesn't work no more.

File: Inc_Heated_Cylinders/flow_cylinder.cfg (hunk 2 above)
Comment (oleburghardt) [reply Inc_Heated_Cylinders/flow_cylinder.cfg]: Yes, I'll change

File: Inc_Heated_Cylinders/flow_cylinder.cfg (hunk 3 above)
Comment (oleburghardt) [reply Inc_Heated_Cylinders/flow_cylinder.cfg]: I'll replace them

File: Inc_Heated_Cylinders/flow_cylinder.cfg (hunk 1 above)
Comment (oleburghardt) [reply Inc_Heated_Cylinders/flow_cylinder.cfg]: Yes, ok.

**Comment by oleburghardt** [comment]
@TobiKattmann Please have a look, it's all corrected now.

File: Inviscid_3D_Constrained_ONERAM6/inv_ONERAM6_adv.cfg [hunk 4]
```diff
@@ -105,7 +105,7 @@ OBJECTIVE_FUNCTION= DRAG
 CFL_NUMBER= 7.5
 %
 % Number of total iterations
-ITER= 1500
+EXT_ITER= 1500
```
Comment (TobiKattmann) [inline Inviscid_3D_Constrained_ONERAM6/inv_ONERAM6_adv.cfg]: This should stay `ITER `right?

Comment (TobiKattmann) [reply Inviscid_3D_Constrained_ONERAM6/inv_ONERAM6_adv.cfg]: I went ahead and changed it back to ITER

**Pull request merged**